homeassistant/components/shelly/utils.py

```diff
@@ -346,3 +346,30 @@ def get_rpc_input_triggers(device: RpcDevice) -> list[tuple[str, str]]:
             triggers.append((trigger_type, subtype))
 
     return triggers
+
+
+async def async_device_update_info(
+    hass: HomeAssistant, shellydevice: BlockDevice | RpcDevice, shellyentry: ConfigEntry
+) -> None:
+    """Update device registry info."""
+
+    _LOGGER.debug("Updating device registry info")
```

Comment:
Let's move this to where we actually update the data (line 370) and add the name of the updated entry
```suggestion
    _LOGGER.debug("Updating device registry info for %s", entry.title)
```